tests/unit_tests/training/test_config.py

```
@@ -1206,6 +1207,161 @@ def test_modelopt_requires_no_gradient_accumulation_fusion(self, monkeypatch):
         finally:
             restore_get_world_size_safe(og_ws, cfg_mod)
 
+    def test_fake_process_group_requires_torch_2_3(self, monkeypatch):
```

Ruff (0.14.14)

Unused method argument: `monkeypatch`

(ARG002)